common/lib/capa/capa/tests/test_input_templates.py

```diff
@@ -290,11 +281,9 @@ def test_problem_marked_incorrect(self):
         (not a particular option) is marked incorrect.
         """
         conditions = [
-            {'status': Status('incorrect'), 'input_type': 'radio', 'value': ''},
```

Comment:
They were deleted and moved to their own test because the expectation changed.  I am not including the status the radio button in the collected indicator status unless the problem is unanswered.  This is to ensure there is only one status span and id.

I still need to look into why the HTML differs.